packages/cli/README.md

```diff
@@ -221,10 +238,95 @@ Commands for managing Bucket apps.
 
 These options can be used with any command:
 
-- `--debug`: Enable debug mode for verbose output
-- `--base-url <url>`: Set the base URL for Bucket API
-- `--api-url <url>`: Set the API URL directly (overrides base URL)
-- `--help`: Display help information for a command
+- `--debug`: Enable debug mode for verbose output.
+- `--base-url <url>`: Set the base URL for Bucket API.
+- `--api-url <url>`: Set the API URL directly (overrides base URL).
+- `--help`: Display help information for a command.
+
+## AI-Assisted Development
+
+Bucket provides powerful AI-assisted development capabilities through rules and Model Context Protocol (MCP). These features help your AI development tools better understand your features and provide more accurate assistance.
+
+### Bucket Rules (Recommended)
+
+The `rules` command helps you set up AI-specific rules for your project. These rules enable AI tools to better understand how to work with Bucket and feature flags and how they should be used in your codebase.
+
+```bash
+npx bucket rules [--format cursor|copilot] [--yes]
+```
+
+Options:
+
+- `--format`: Format to add rules in:
+  - `cursor`: Adds rules to `.cursor/rules/bucket.mdc` for Cursor IDE integration.
+  - `copilot`: Adds rules to `.github/copilot-instructions.md` for GitHub Copilot integration.
+- `--yes`: Skip confirmation prompts and overwrite existing files without asking.
+
+This command will add rules to your project that provide AI tools with context about how to setup and use Bucket feature flags. For the copilot format, the rules will be added to a dedicated section in the file, allowing you to maintain other copilot instructions alongside Bucket's rules.
+
+## Model Context Protocol
+
+The Model Context Protocol (MCP) is an open protocol that provides a standardized way to connect AI models to different data sources and tools. In the context of Bucket, MCP enables your development environment to understand your feature flags, their states, and their relationships within your codebase. This creates a seamless bridge between your feature management workflow and AI-powered development tools. MCP is in a very early stage of development and changes are frequent, if something isn't working please check out the [Model Context Protocol Website](https://modelcontextprotocol.io/) and open an [issue ticket here](https://github.com/bucketco/bucket-javascript-sdk/issues).
+
+### Setting up MCP
+
+MCP servers currently run locally on your machine. To start the MCP server run the CLI command from your Bucket initialized project directory:
+
+```bash
+npx bucket mcp [--port <number|"auto">] [--app-id ap123456789]
```

Comment:
given that users will have a `bucket.config.json` file with `appId` in it, it's not required to pass it here, right?

Reply:
It is not required denoted by the `[...]` brackets 😊

Reply:
okay, i think we should omit it here then to reduce confusion as per our mantra on opinionated approach with one way to achieve things. i.e. always just use the `bucket.config.json` file and avoid overlapping CLI args 

Cc. @roncohen

Reply:
While I agree with you that we should have an opinionated way of doing things and be as frictionless as possible, I disagree with the idea that the way to achieve that is by removing choice. Choice, I feel, is especially important for a developer-centric tool such as a CLI. Finally, I can also see value in being able to start the MCP server outside of a project by calling `npx bucket mcp --appId ap12345`.

Reply:
There's a difference between reference docs and getting started guides. In addition to helping
folks get started, the Getting started guide is also what you use to understand as a user if you want to invest in going through it. So, the getting started guides _need to be as simple as possible_ while the reference can show all the possible options. When writing docs we have to be aware of whether we're writing the getting started part or the reference. In this example, is it a reference or getting started? That will answer whether we should include the parameter or not.